code/modules/clothing/factions/suns.dm

```diff
@@ -0,0 +1,572 @@
+	desc = "A sterile mask designed to help prevent the spread of diseases. Now in purple! Pretty!"
+	icon_state = "suns_sterile"
+	item_state = "suns_doctorcap"
+	icon = 'icons/obj/clothing/faction/suns/mask.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/mask.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/mask/breath/suns
+	name = "\improper SUNS half face mask"
+	desc = "A close-fitting mask that covers JUST enough to connect an air supply."
+	icon_state = "suns_captainmask"
+	item_state = "suns_captainmask"
+	icon = 'icons/obj/clothing/faction/suns/mask.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/mask.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+
+///////////
+//Gloves//
+//////////
+
+
+/obj/item/clothing/gloves/color/captain/suns
+	name = "\improper SUNS captain's gloves"
+	desc = "Fancy black gloves for trusted SUNS members. Sports a complex lining that prevents the wearer from being shocked."
+	icon_state = "suns_captaingloves"
+	item_state = "suns_blackgloves"
+	icon = 'icons/obj/clothing/faction/suns/hands.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/hands.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/gloves/color/sunsxo
+	name = "academic staff gloves"
+	desc = "White gloves that offer a good grip with writing utensils."
+	icon_state = "suns_xogloves"
+	item_state = "suns_whitegloves"
+	icon = 'icons/obj/clothing/faction/suns/hands.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/hands.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/gloves/fingerless/suns
+	name = "stitched fingerless gloves"
+	desc = "These gloves offer style, purely and plainly."
+	icon_state = "suns_glovesfingerless"
+	item_state = "suns_blackgloves"
+	icon = 'icons/obj/clothing/faction/suns/hands.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/hands.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/gloves/color/yellow/suns
+	name = "insulated gloves"
+	desc = "Padded academic gloves that hopefully keep students out of the nurses office."
+	icon_state = "suns_insulated"
+	item_state = "suns_blackgloves"
+	icon = 'icons/obj/clothing/faction/suns/hands.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/hands.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/gloves/color/latex/nitrile/evil/suns
```

Comment:
Any reason it's `/evil` subtype? I don't think those have any special behavior that warrants it.

Reply:
Holdover from previous PR